### DISCUSSION TITLE
如何正确排版双栏算法

### DISCUSSION THREAD

**Discussion by koperia** [note286/xduts#92] [Q&A] [answered]
开发者你好，我有排版双栏算法的需求，我的一个案例如下

但是这两处间距不知怎么才能去掉，望解答，谢谢！
![image](https://user-images.githubusercontent.com/67058191/227130549-38a8a3f0-248b-4954-8756-cdf8bb9211a7.png)

**Comment by note286** [comment, marked as answer]
[spacing - How to get rid of extra space after multicol environment? - TeX - LaTeX Stack Exchange](https://tex.stackexchange.com/questions/400067/how-to-get-rid-of-extra-space-after-multicol-environment)

**Reply by koperia** [reply]
谢谢，用 `\vspace{-0.3cm}` 微调下效果可以了
